canonical-web-and-design/maas.io
# Maas docs

The new docs section - maas.io/docs - to replace docs.maas.io.

## QA

Get hold of the "Custom Search API key" from [the Google cloud API credentials page](https://console.cloud.google.com/apis/credentials?project=ubuntu-search-1530889417216) - if you don't have access to that page, ask @nottrobin or someone for access.

Then, run the site with:

``` bash
./run --env SEARCH_API_KEY={the-api-key}
```

Now go to http://0.0.0.0:8006/docs, check it looks good, click around, check links work etc.
